### PR TITLE
dateBR Validation

### PR DESCRIPTION
**Pull request by clebersleite**
#### Description
Brazilian date validation to format: dd/mm/yyyy

### CODE REVIEW

File: src/additional/dateBR [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// Brazillian Date validation
+// Rule: dd/MM/YYYY
+
```
Comment (staabm) [inline src/additional/dateBR]: misses proper docs

File: src/additional/dateBR [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+                    var mes         = data.substr(3,2);         
+                    var barra2      = data.substr(5,1);
+                    var ano         = data.substr(6,4);         
+                    if(data.length!=10||barra1!="/"||barra2!="/"||isNaN(dia)||isNaN(mes)||isNaN(ano)||dia>31||mes>12) return false; 
+                    if((mes==4||mes==6||mes==9||mes==11) && dia==31) return false;
+                    if(mes==2 && (dia>29||(dia==29 && ano%4!=0))) return false;
+                    if(ano < 1900)  return false;
+                    
+                    return true;        
+}, "Informe uma data válida");
```
Comment (staabm) [inline src/additional/dateBR]: should use translation mechanism, see other date* implementations

**Comment by staabm** [comment]
also misses a proper PR description.

please give your PR more love, in case you really want it be part of this project.

**Comment by clebersleite** [comment]
Aguardando aprovação....

**Comment by clebersleite** [comment]
Waiting Approved...

**Pull request closed without merging**